matt-nate/server.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('./public'));
+
```

Comment:
well want to add and app.get here at /new to serve up new.html with the res.sendFile method